src/VirtoCommerce.DescriptionExportImportModule.Data/Services/DescriptionExportPagedDataSource.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.CatalogModule.Core.Services;
+using VirtoCommerce.DescriptionExportImportModule.Core.Models;
+using VirtoCommerce.DescriptionExportImportModule.Core.Services;
+
+namespace VirtoCommerce.DescriptionExportImportModule.Data.Services
+{
+    public class DescriptionExportPagedDataSource : IDescriptionExportPagedDataSource
```

Comment:
Why naming does not match the domain model's name? I mean EditorialReview or just Review.